.github/workflows/qcom-preflight-checks-reusable-workflow.yml

```diff
@@ -131,3 +150,16 @@ jobs:
           base-ref: ${{ steps.setrefs.outputs.base_ref }}
           head-ref: ${{ steps.setrefs.outputs.head_ref }}
           fail-on-severity: high  
+
+  commit-msg-check:
+    if: ${{ inputs.commit-msg-check && github.event_name == 'pull_request_target' }}
```

Comment:
Why do we have to use pull_request_target? I'd rather not use it. What secrets does it need access to?

Reply:
Yeah but we also need it if we are going to use secrets.GITHUB_TOKEN, which I'd rather avoid.